anasos-kernel/build.rs

```diff
@@ -0,0 +1,45 @@
+use std::env;
+use std::process::Command;
+
+fn main() {
+    let target_dir: String = env::var("OUT_DIR").unwrap(); 
+
+    let boot_o_path: String = format!("{}/boot.o", target_dir);
+    let boot_64_o_path: String = format!("{}/boot-64.o", target_dir);
+    let header_o_path = format!("{}/header.o", target_dir);
+
+
+    let status_boot = Command::new("nasm")
```

Comment:
Please verify that the Release GitHub Actions workflow has all the tools required already configured before merging this as the workflow may fail.